src/Lykke.Service.BlockchainApi.Sdk/Controllers/TransactionsController.cs

```diff
@@ -34,15 +40,27 @@ public class TransactionsController : ControllerBase
             _log = logFactory.CreateLog(this);
         }
 
-        private async Task<ActionResult<BuildTransactionResponse>> Build(Guid operationId, string assetId, 
-            (string from, string fromContext, string to, string amount)[] inOuts, bool includeFee)
+        private async Task<ActionResult<BuildTransactionResponse>> Build(
+            Guid operationId, 
+            string assetId, 
+            (string from, string fromContext, string to, string amount)[] inOuts,
+            bool includeFee)
         {
             var operation = await _operations.GetAsync(operationId);
 
-            if (operation != null && 
-                operation.IsRunning()) 
+            if (operation != null)
             {
-                return Conflict($"Operation is already {operation.GetState()}");
+                if (operation.FailTime != null)
```

Comment:
This is not clear from documentation, but for the Build Conflict should be returned if Broadcast was executed despite of its execution result. So, we need to restore previous version of this code

Reply:
Previous logic restored.